mobile/Screen/ArtworkDetailScreen.js

```diff
@@ -275,6 +278,10 @@ function ArtworkDetailScreen({
     setShowFullDescription(!showFullDescription);
   };
 
+  React.useLayoutEffect(() => {
```

Comment:
As opposed to this, it might be better to use the `header shown` option in the Stack Navigator (as in https://github.com/lablueprint/skid-row-arts-alliance/blob/art-details-screen/mobile/Components/AppNavigator.js#L187); this will delete the entire header and allow you to create a custom one with the arrow (like in the hi-fi/final design)